src/core/sys/linux/timerfd.d

```diff
@@ -0,0 +1,24 @@
+/**
+ * D header file to interface with the Linux timefd API <http://man7.org/linux/man-pages/man2/timerfd_create.2.html>
+ * Available since Linux 2.6
+ *
+ * License : $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ */
+module core.sys.linux.timerfd;
+
+version (linux):
+
+public import core.sys.posix.time;
```

Comment:
On Sat, Sep 12, 2015 at 03:41:57AM -0700, Dmitry Olshansky wrote:
> public imports are dangerous ;) There is at least one bug with them still open

Perhaps we should fix that bug!

I could make it private, but the file is basically useless without that
itimerspec definition, and the file is straight up #included in the C
source so bugs notwithstanding, the public import is the closest match.